### PR TITLE
[FIX]comment pylint avoid message error by different apis

### PR DESCRIPTION
**Pull request by jorgescalona**
travis warning disable by different api error & duplicate method

### CODE REVIEW

File: default_warehouse_from_sale_team/models/sales_team.py [hunk 1]
```diff
@@ -105,6 +105,8 @@ def read(self, cr, user, ids, fields_list=None, context=None,
         return super(WarehouseDefault, self).read(
             cr, user, ids, fields=fields_list, context=context, load=load)
 
+    # pylint: disable=E0102
```
Comment (moylop260) [inline default_warehouse_from_sale_team/models/sales_team.py]: Please, use the name of the error

Comment (jorgescalona) [reply default_warehouse_from_sale_team/models/sales_team.py]: done

**Comment by JulioSerna** [comment]
why the messages commit?

> [cualquiercosa]

**Comment by JulioSerna** [comment]
@jorgescalona 

I am going to close this PR is better create other new:
1. Create new branch dev do not re-use branch dev in this case you used the branch dev of other PR already merged
2. Create branch dev updated with base repository

**Pull request closed without merging**